.github/workflows/catalyst.yaml

```diff
@@ -81,10 +91,11 @@ jobs:
         with:
```

Comment:
I don't understand if we push entire `Dockerfile` results into both `livepeerci/in-a-box` and `livepeerci/catalyst`?
Or `catalyst` is stripped to `catalyst` stage in [Dockerfile](https://github.com/livepeer/catalyst/pull/590/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R53-R81)

Reply:
Never mind. I see that `target:` does stage selection https://docs.docker.com/engine/reference/commandline/build/#target